src/Hit/Issue.hs

```diff
@@ -93,6 +95,18 @@ getIssueTitle num = fetchIssue num >>= \case
     Left err -> errorMessage (show err) >> exitFailure
     Right Issue{..} -> pure issueTitle
 
+assignToIssue :: Id Issue -> IO (Either Error Issue)
+assignToIssue iNum = withOwnerRepo action
+  where
+    action Nothing _ _ = pure $ Left $ UserError "GITHUB_TOKEN lookup failed"
+    action (Just t) o r = issue' (Just t) o r iNum >>= \case
+        Left e -> return $ Left e
+        Right i -> do
+            let currentAssignees = simpleUserLogin <$> issueAssignees i
+                newAssignee = makeName . untagName $ o
```

Comment:
Is there any reason to prefer `makeName . untagName $ o` over shorter `makeName $ untagName o`? I see that the style of `f . g . h $ x` is common Haskell, but I personally find it awkward. It requires you to type more characters for no benefit.